conference-demos/onnxruntime-openvino-lcm-demo/README.md

```diff
@@ -0,0 +1,52 @@
+# Image generation using Latent Consistency Model and ONNX Runtime with OpenVINO Execution Provider 
+
+LCMs: The next generation of generative models after Latent Diffusion Models (LDMs). 
+Latent Diffusion models (LDMs) have achieved remarkable results in synthesizing high-resolution images. However, the iterative sampling is computationally intensive and leads to slow generation.
+
+**Input text:** tree with lightning in the background, 8k
+
+<p align="center">
+    <img src="https://github.com/openvinotoolkit/openvino_notebooks/assets/105707993/73cb12e3-152d-463a-bb06-5ea0ddedc6d6"/>
+</p>
+
+### Notebook Contents
+
+This [notebook](./Latent Consistency Models with ONNX and OpenVINO Execution Provider.ipynb) demonstrates how to  run [LCM_Dreamshaper_v7](https://huggingface.co/SimianLuo/LCM_Dreamshaper_v7) using ONNX Runtime and OpenVINO Execution Provider on iGPU of AI PC 
+
+
+
+## Installation Instructions
+- We recommend to use virtual environment. To create it use python -m venv <virtual-environment-name>
```

Comment:
I wouldn't use optional recommendations. Just say "create a virtual env". It's easier to follow

Reply:
fixed